relay-server/src/actors/store.rs

```diff
@@ -649,6 +649,17 @@ impl Handler<StoreEnvelope> for StoreForwarder {
                 attachments,
             });
 
+            if self
+                .config
+                .processing_internal_projects()
+                .contains(&scoping.project_id.value())
+            {
+                metric!(
+                    counter(RelayCounters::InternalCapturedEventStoreActor) += 1,
```

Comment:
At this place, you only observe accepted transactions. In the PR description, you talk about dropped transactions -- if we drop them unintentionally you'd probably also have to capture another metric in `endpoints/common`.

Reply:
ive added one in the endpoint where I think it may make sense, but I am really not sure